src/main/java/com/hillel/items_exchange/security/jwt/JwtTokenProvider.java

```diff
@@ -60,20 +63,25 @@ private String getUsername(String token) {
 
     public String resolveToken(HttpServletRequest req) {
         String bearerToken = req.getHeader(AUTHORIZATION_HEADER_NAME);
-        if (bearerToken != null && bearerToken.startsWith(BEARER_PREFIX)) {
-            return bearerToken.substring(7);
+        if(bearerToken != null) {
+            if(bearerToken.startsWith(BEARER_PREFIX)) {
+                return bearerToken.substring(7);
+            } else {
+                log.error("Unauthorized: {}", messageSource.getMessage("token.not.start.with.bearer", null, Locale.US));
```

Comment:
You should extract ``messageSource.getMessage("token.not.start.with.bearer", null, Locale.US))`` to a new variable here and to use default locale.

Reply:
Fixed